test/cmd-collect-exit.test.js

```diff
@@ -47,7 +47,15 @@ test('cmd - collect - non-success exit code should not throw', function (t) {
   })
 })
 
-test('cmd - collect - non-success exit code causes error', function (t) {
+test('cmd - collect - SIGKILL causes error', function (t) {
+  // On Windows, process.exit(1) and'process.kill(pid, 'SIGKILL') both give exit code 1
+  // TODO: test Windows SIGKILL behaviour, if it should error, find reliable detection
+  if (os.platform() === 'win32') {
+    t.pass('Skip test as SIGKILL also sends exit code 1 on windows')
+    t.end()
```

Comment:
there is a { skip } option that you can pass to test as 2nd parameter

Reply:
I was matching how the existing test does it - { skip } is probably better though so I've updated both